### ISSUE TITLE
i3blocks will not accept input from stdout:w after too long of inactivity

### ISSUE THREAD

**Issue by NateDogg1232** [vivien/i3blocks#450]
If there's too much time between outputs to stdin for the block, i3blocks will no longer output stdin.

**Comment by NateDogg1232** [comment]
Sorry about the title edit. I was just using vim before opening this issue and tried saving the title. Whoops!

**Comment by zsugabubus** [comment]
Cannot be that it only occurs with `xtitle -s`? Have you tested others? (Maybe a simple `echo && sleep && echo`?)

Just because if you search the issue tracker you will see a few reports about `xtitle`.

**Comment by NateDogg1232** [comment]
> Cannot be that it only occurs with xtitle -s? Have you tested others? (Maybe a simple echo && sleep && echo?)

Yeah I'm gonna try that overnight. It's very late here so I'm going to bed now, but I'll run multiple tests like this.
The tests I'll be running:
```ini
[test]
command=xtitle -s
interval=persist
```
```ini
[test]
command=while true; do echo "Hello" && sleep 512 && echo "Hello2" && sleep 512; done
interval=persist
```
```ini
[test]
command=while true; do echo "Hello" && sleep 2048 && echo "Hello2" && sleep 2048; done
interval=persist
